File: truth_set/python/structure/solution-approach/semantic-duplication/utils/dateUtils.py
def format_date(date, format='YYYY-MM-DD'):
    year = date.year
    month = f"{date.month:02d}"
    day = f"{date.day:02d}"
    
    if format == 'MM/DD/YYYY':
        return f"{month}/{day}/{year}"
    elif format == 'DD-MM-YYYY':
        return f"{day}-{month}-{year}"
    else:
        return f"{year}-{month}-{day}"
\ No newline at end of file

File: truth_set/go/structure/solution-approach/semantic-duplication/main/utils/dateUtils.go
package utils

import "time"

func FormatDate(date time.Time, format string) string {
	year := date.Format("2006")
	month := date.Format("01")
	day := date.Format("02")

	switch format {
	case "MM/DD/YYYY":
		return month + "/" + day + "/" + year
	case "DD-MM-YYYY":
		return day + "-" + month + "-" + year
	default:
		return year + "-" + month + "-" + day
	}
}
\ No newline at end of file